### PR TITLE
Integration/portable migration

### PR DESCRIPTION
**Pull request by ssipasseuth**
Reimplemented the life cycle of PIC according to https://oat-sa.atlassian.net/browse/TAO-2524

Requires https://github.com/oat-sa/extension-tao-itemqti/pull/765
Please check this related PR for more information.

### CODE REVIEW

File: views/js/picProvider.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    
+    return {
+        load: function load(){
+            return new Promise(function(resolve){
+                $.ajax({
+                    url : _serverUrl,
+                    dataType : 'json',
+                    type : 'GET'
+                }).done(function (pcis){
+                    resolve(pcis);
```
Comment (jsconan) [inline views/js/picProvider.js]: What happens if the query fails ? I assume the promise could be never resolved...

File: views/js/qtiCreator/plugins/panel/picManager.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+define([
```
Comment (jsconan) [inline views/js/qtiCreator/plugins/panel/picManager.js]: No header ?

**Comment by ssipasseuth** [comment]
I have fixed the issues, please take a look again.

**Comment by siwane** [comment]
This PR will be replace by branch https://github.com/oat-sa/extension-tao-itemqti-pic/tree/feature/portableElement

**Pull request closed without merging**